subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -602,17 +602,40 @@ Gradle will behave as if you ran `gradle test` after a change to sources or test
 This means that unrelated changes (such as changes to build scripts) will not trigger a rebuild.
 In order to incorporate build logic changes, the continuous build must be restarted manually.
 
+Continuous build uses file system watching to detect changes to the inputs.
+If file system watching does not work on your system, then continuous build won't work either.
+Especially, continuous build does not work when using `--no-daemon`.
+See <<gradle_daemon.adoc#sec:daemon_watch_fs,Watching the file system>> for more information.
+
 === Terminating Continuous Build
 
 If Gradle is attached to an interactive input source, such as a terminal, the continuous build can be exited by pressing `CTRL-D` (On Microsoft Windows, it is required to also press `ENTER` or `RETURN` after `CTRL-D`).
 If Gradle is not attached to an interactive input source (e.g. is running as part of a script), the build process must be terminated (e.g. using the `kill` command or similar).
 If the build is being executed via the Tooling API, the build can be cancelled using the Tooling API's cancellation mechanism.
 
 [[continuous_build_limitations]]
-=== Limitations and quirks
+=== Limitations
+
+Under some circumstances continuous build will not detect changes to inputs.
+
+[[sec:continuous_build_missing_files]]
+==== Adding new root input directories
+
+Sometimes, adding root input directories does not trigger a build, due to the way file system watching works.
+For example, when adding the `src/main/java` directory, Gradle may not trigger a build.
+Similarly, if the input is a <<working_with_files.adoc#filtering_files,filtered file tree>>, Gradle may not detect the addition of the first file matching the filter.
+
+[[sec:continuous_build_untracked]]
+==== Inputs of untracked tasks
+
+Gradle will only watch for changes of inputs of tasks if it captures their state.
+That means changes to the inputs of <<more_about_tasks.adoc#sec:disable-state-tracking,untracked tasks>> or tasks that have no outputs will not trigger a build.
```

Comment:
I would either change these two sentences around, or skip the first one completely. The important bit for the user is that untracked and no-output tasks won't trigger builds.

Reply:
I skipped the first sentence.